src/main/scala/reactive/Main.scala

```diff
@@ -0,0 +1,96 @@
+package reactive
```

Comment:
I always follow the Java convention of having "reverse url" package names in Scala. Is it a conscious decision not to do that here? Does Typesafe have any opinions on this kind of thing in the context of Activator templates?